### PR TITLE
ES2017 review, second round

### PR DESCRIPTION
**Pull request by anba**
Mostly fixes nits in the Atomics and memory model chapters.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -32180,16 +32211,23 @@
...
             1. Else, let _srcByteIndex_ be _srcByteOffset_.
             1. Let _targetByteIndex_ be _targetOffset_ &times; _targetElementSize_ + _targetByteOffset_.
             1. Let _limit_ be _targetByteIndex_ + _targetElementSize_ &times; _srcLength_.
-            1. If both IsSharedArrayBuffer(_srcBuffer_) and IsSharedArrayBuffer(_targetBuffer_) are *true*, and _srcBuffer_.[[ArrayBufferData]] is equal to _targetBuffer_.[[ArrayBufferData]], then let _same_ be *true*, else let _same_ be SameValue(_srcBuffer_, _targetBuffer_).
+<<<<<<< HEAD
+            1. If SameValue(_srcType_, _targetType_) is *true*, then
+=======
+            1. If both IsSharedArrayBuffer(_srcBuffer_) and IsSharedArrayBuffer(_targetBuffer_) are *true*, and _srcBuffer_.[[ArrayBufferData]] is equal to _targetBuffer_.[[ArrayBufferData]], let _same_ be *true*, else let _same_ be SameValue(_srcBuffer_, _targetBuffer_).
             1. If _same_ is *true*, then
+>>>>>>> e90f50b... Remove 'then' in single-line if-steps
```
Comment (azu) [inline spec.html]: conflict?

Comment (anba) [inline spec.html]: Thanks, probably happened while reordering the commits. Will fix and rebase.

Comment (anba) [inline spec.html]: force rebased to remove the conflict marker

**Comment by bterlson** [comment]
I've pulled all these in (ordered, rebased, minus the double-a commit which was already fixed). Find the last commit here: 581023c.

**Pull request closed without merging**